config/locales/client.en.yml

```diff
@@ -1861,6 +1862,7 @@ en:
       tags_allowed_tag_groups: "Tag groups that can only be used in this category:"
       tags_placeholder: "(Optional) list of allowed tags"
       tag_groups_placeholder: "(Optional) list of allowed tag groups"
+      topic_featured_link_allowed: "Only allow posting a featured link with topics in the enabled category"
```

Comment:
Rephrased as "Restricts editing the topic featured link in this category. Require site setting topic_featured_link_enabled is checked."